cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/DefaultBrokerSetAssignmentPolicy.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.config;
+
+import com.linkedin.kafka.cruisecontrol.exception.BrokerSetResolutionException;
+import com.linkedin.kafka.cruisecontrol.model.ClusterModel;
+import java.util.Map;
+import java.util.Set;
+import org.apache.kafka.common.annotation.InterfaceStability;
+
+
+/**
+ * For the Brokers that do not have a BrokerSet assigned, this policy will determine the BrokerSet
+ * assignment
+ */
+@InterfaceStability.Evolving
+public interface DefaultBrokerSetAssignmentPolicy {
```

Comment:
Since it's an interface, it feels strange to have "Default" in the name, as there's no default implementation provided within the interface. BrokerSetAssignmentPolicy sounds better to me.

Reply:
I was initially thinking that this is the behavior where unmapped brokers would be "defaulted" to certain brokerSet. However, this works if none of the brokers are mapped too. So yeah, making the name change makes sense.